stock_available/__openerp__.py

```diff
@@ -4,7 +4,7 @@
 
 {
     'name': 'Stock available to promise',
-    'version': '9.0.1.0.0',
+    'version': '9.0.3.0.0',
```

Comment:
I don't think so. It's not the same. We have reset the version number to 9.0.1.0.0, so the documentation is correct, and you have to increment x, y or z according this new version, not according 8.0 version.